orbit/cmd/orbit/orbit.go

```diff
@@ -121,7 +121,7 @@ func main() {
 		},
 		&cli.DurationFlag{
 			Name:    "update-interval",
-			Usage:   "How often to check for updates",
+			Usage:   "How often to check for updates. Note: fleetd checks for updates at startup. The next update check adds some randomization and may take up to 10 minutes longer",
```

Comment:
```suggestion
			Usage:   "How often to check for updates. Note: fleetd checks for updates at startup. Before the first update check a random wait interval is set to prevent agents thundering (may take up to 10 minutes longer)",
```